### PR TITLE
[PfemFluid] Assign and transfer processes into the PfemFluid

### PR DESCRIPTION
**Pull request by AFranci**
Now the PfemFluid should work without the Solid app

### CODE REVIEW

**Comment by ipouplana** [comment]
@AFranci did you checked that the SDEM tests work ? (especially the PFEM-DEM test)

File: applications/DelaunayMeshingApplication/python_scripts/domain_utilities.py [hunk 1]
```diff
@@ -160,9 +160,9 @@ def AddBoundaryNodesToFluidDomains(self,model_part):
             print("::[--Domain Utilities-]:: Add boundary nodes to fluid domains ")
```
Comment (rubenzorrilla) [inline applications/DelaunayMeshingApplication/python_scripts/domain_utilities.py:160]: Just wondering if the changes in this file might affect other users of the `DelaunayMeshingApplication`.

Comment (AFranci) [reply applications/DelaunayMeshingApplication/python_scripts/domain_utilities.py:160]: @rubenzorrilla thanks for the observation!
I think that nobody but PfemFluid people can use that function, anyway I've now moved the transfer process from the PfemFluid to Delaunay Application to resolve possible conflicts.

**Comment by AFranci** [comment]
@ipouplana, yes, all tests are running well!

**Review by ipouplana — APPROVED** [review]

**Pull request merged**